nexus/src/external_api/http_entrypoints.rs

```diff
@@ -4023,6 +4024,97 @@ async fn updates_refresh(
     apictx.external_latencies.instrument_dropshot_handler(&rqctx, handler).await
 }
 
+// Metrics
+
+#[derive(Debug, Deserialize, JsonSchema)]
```

Comment:
This name seems confusing...it doesn't seem to be a resources or a utilization metric?  I gather it's the query parameter used to paginate the list of data points for a particular resource's metrics (whether they're utilization metrics or not)?  Maybe `ResourceMetricPaginationParams` (looking to `IpPoolRangePaginationParams` for a precedent -- the rest of the examples are `PaginatedByX`)

Reply:
Just seems like an oversight. I wouldn’t put pagination in the name, that makes it sound like it’s _only_ for that. `SystemMetricParams` or even `QueryParams` would be better.

Reply:
Updated to `SystemMetricParams`: 8d153f6277e82436363181377266760c72356e50